hash/gbcolor.xml

```diff
@@ -7453,6 +7453,52 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="gberetp3">
+		<!-- Notes: GBC only -->
+		<description>Green Beret (prototype 3)</description>
+		<year>2000</year>
+		<publisher>ClearWater Interactive</publisher>
+		<part name="cart" interface="gameboy_cart">
+			<feature name="slot" value="rom_mbc5" />
+			<dataarea name="rom" size="262144">
+				<rom name="green beret (prototype 3).bin" size="262144" crc="fac3f4c5" sha1="dd69f3d9ef87198334b4cc0f54c88a29fb353e44"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="gberetp1" cloneof="gberetp3">
+		<!-- Notes: GBC only -->
+		<!--	Fixed to work on real hardware by replacing "C0" to "E0" at offset 0x153.	-->
```

Comment:
I don’t think “fixed” images like this should be included in software lists – if it didn’t work on hardware, then that’s the way it should really be preserved.  People can use cheats, etc. to apply “fixes”.

Reply:
There you go, unfixed so that they won't work.